fvm/errors.go

```diff
@@ -198,6 +200,25 @@ func (e *InvalidProposalKeyMissingSignatureError) Code() uint32 {
 	return errCodeInvalidProposalKeyMissingSignature
 }
 
+// MaxEventSizeLimitExceededError indicates that the transaction has produced events with size more than limit.
+type MaxEventSizeLimitExceededError struct {
+	TotalByteSize uint64
+	Limit         uint64
+}
+
+func (e *MaxEventSizeLimitExceededError) Error() string {
+	return fmt.Sprintf(
+		"The total byte size of events is %d which is more than the limit %d .",
```

Comment:
Go pattern is to keep error message short and concise, always using lowercase 

> total event byte size (%d) exceeds limit (%d)

Reply:
Applied at 5167714